queries.sql

```diff
@@ -70,4 +70,13 @@ SELECT MIN(weight_kg) FROM animals GROUP BY species;
 SELECT MAX(weight_kg) FROM animals GROUP BY species;
 
 /* What is the average number of escape attempts per animal type of those born between 1990 and 2000? */
-SELECT AVG(escape_attempts) FROM animals GROUP BY species, date_of_birth HAVING (date_of_birth) BETWEEN '1990-01-01' AND '2000-01-01';
\ No newline at end of file
+SELECT AVG(escape_attempts) FROM animals GROUP BY species, date_of_birth HAVING (date_of_birth) BETWEEN '1990-01-01' AND '2000-01-01';
+
+/* Milestone 3 */
+SELECT name FROM animals INNER JOIN owners ON animals.owners_id = owners_id WHERE owners.full_name = 'Melody Pond';  --What animals belong to Melody Pond?
```

Comment:
- [ ] I think we should try to do the same for all other comments in all files by removing the comments. 👍